### DISCUSSION TITLE
Running production code in the kitchen-sink/apps/api example

### DISCUSSION THREAD

**Discussion by pstachula-dev** [vercel/turborepo#443] [Help] [answered]
https://github.com/vercel/turborepo/blob/e5b6baabcbaed396de6d0d28118f3b02c4fdff0d/examples/kitchen-sink/apps/api/package.json#L8

Why the production command uses codes from src folder instead of dist?

**Comment by jaredpalmer** [comment, marked as answer]
[`esbuild-register`](https://github.com/egoist/esbuild-register) is a CLI tool that runs `esbuild` on the fly, so there is no "dist" folder at all

**Reply by pstachula-dev** [reply]
Wow it has to be brilliant tool, really this enough performant formant for production usage?
